src/SAWScript/Crucible/Common/MethodSpec.hs

```diff
@@ -15,53 +15,137 @@ Grow\", and is prevalent across the Crucible codebase.
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE StandaloneDeriving #-}
 {-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE TypeFamilyDependencies #-}
 {-# LANGUAGE UndecidableInstances #-}
 
-module SAWScript.Crucible.Common.MethodSpec where
+module SAWScript.Crucible.Common.MethodSpec
+  ( AllocIndex(..)
```

Comment:
Note to reviewers: this module still exports the same things that it did before this patch. I added an explicit export list here because this module now re-exports some things from `SAWScript.Crucible.Common.Setup.Value`, which requires using an export list to accomplish.